src/building-blocks/Info.tsx

```diff
@@ -8,25 +8,41 @@ import { ClassValue } from 'classnames/types';
 import * as Antd from 'antd';
 
 import { CLASS_PREFIX } from '../consts';
-import { IFieldConfigPartial } from '../interfaces';
+import { IFieldConfigPartial, IFormatProps } from '../interfaces';
 
 @autoBindMethods
 @observer
-class Info extends Component<{ fieldConfig: IFieldConfigPartial }> {
+class Info extends Component<{ fieldConfig: IFieldConfigPartial; format?: IFormatProps }> {
   public render() {
+    const { format } = this.props,
+      layout = format && format.layout,
+      rowClassName = layout !== 'inline' ? '' : `${CLASS_PREFIX}-info-item-inline`;
```

Comment:
Why only include a class sometimes?
```suggestion
      rowClassName = `${CLASS_PREFIX}-info-row-${layout}`;
```

(actually asking)

Reply:
I guess this was because I didn't need any styles for the non-inline version of the row, but that makes sense that there should probably be a class name regardless

Reply:
Yeah best to think of it as a general-purpose library, if only to save ourselves from having to do releases just to add a class when writing CSS elsewhere.